.grit/patterns/jest_to_vitest.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Jest to Vitest
+---
+
+Convert Jest test to Vitest.
+
+tags: #migration, #js
+
+```grit
+engine marzano(0.1)
+language js
+
+pattern updated_imports() {
+  call_expression($function) where {
+    $function <: or {
+      js"test",
```

Comment:
Jest also has variants like `test.skip` that we need to cover. I think it might be better to just hoist this `or` to the root (skip the `call_expression` part).

Reply:
Thanks @morgante I updated pattern to cover `test.skip` cases. I also add it into pattern test cases.